### PR TITLE
Add fail test case for session_exist_test

### PR DESCRIPTION
**Pull request by Wenfeng-GAO**
Add fail test case for bug report issue:  #1043

### CODE REVIEW

File: session_exist_test.go [hunk 1]
```diff
@@ -73,4 +73,8 @@ func TestExistStruct(t *testing.T) {
 	has, err = testEngine.Table("record_exist").Where("name = ?", "test2").Exist()
 	assert.NoError(t, err)
 	assert.False(t, has)
+
+	has, err = testEngine.Table("record_exist").ID(100).Where("name = ?", "test1").Exist()
+	assert.NoError(t, err)
```
Comment (lunny) [inline session_exist_test.go]:
```Go
assert.Error(t, err)
```

Comment (Wenfeng-GAO) [reply session_exist_test.go]: `Table("record_exist")` with `ID` won't be supported?

Comment (lunny) [reply session_exist_test.go]: Yes, xorm don't know which column is primary key column since no struct was indicated.

**Comment by lunny** [comment]
The current implementation ignored the `ID` function

**Comment by lunny** [comment]
Please feel free to reopen it.

**Pull request closed without merging**